src/vs/workbench/contrib/welcome/page/browser/welcomePage.css

```diff
@@ -3,59 +3,60 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.monaco-workbench .part.editor > .content .welcomePageContainer {
```

Comment:
all of these changes should be reverted.

Reply:
these should be good now